doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -123,7 +124,6 @@ The tools are currently assigned to Tiers as follows:
 
  | Tool Type | Tool/API Name             | Regular Testing in Node.js CI | Integrated with Node.js | Target Tier |
  |-----------|---------------------------|-------------------------------|-------------------------|-------------|
- | FFDC      | node-report               | No                            | No                      |     1       |
```

Comment:
I would rather either keep the npm `node-report` module as unclassified or moved to tier 4.

Reply:
I would move it to tier unclassified.

Reply:
@richardlau @mhdawson - reverted the case of `node-report`, PTAL!